README.md

```diff
@@ -1,4 +1,10 @@
 # Solo.io APIs
 This repository hosts the Solo.io API definitions. It is intended as a read-only mirror; the source of truth for the 
 APIs are the projects that own them. The goal of this repository is to allow for projects to consume an API 
-without having to depend on the project that owns it.
\ No newline at end of file
+without having to depend on the project that owns it.
+
+## Gloo Apis
+Syncing the Gloo APIs is currently a manual process and can be completed with 2 extra steps 
+before running `make generate`
+1. Ensure gloo is checked out to master and available at `../`
+2. run `./hack/sync-apis.sh`
```

Comment:
Want to include suggestions for next steps here too? i.e. (3) create a PR, (4) tag master with the version of gloo that this is relevant for

Reply:
https://github.com/solo-io/solo-apis/issues/4